newIDE/app/src/MainFrame/index.js

```diff
@@ -2627,6 +2633,21 @@ const MainFrame = (props: Props) => {
           if (!canProjectBeSafelySavedAs) return;
         }
 
+        let originalProjectUuid = null;
+        if (newSaveAsOptions && newSaveAsOptions.generateNewProjectUuid) {
+          originalProjectUuid = currentProject.getProjectUuid();
+          currentProject.resetProjectUuid();
```

Comment:
just double checking this will not affect the initial project? I assume not as we're not saving that project

Reply:
In my tests everything went fine, but I'll check again

Reply:
I confirm it's okay